terraform/providers/aws/go.mod

```diff
@@ -2,80 +2,96 @@ module github.com/openshift/installer/terraform/providers/aws
 
 go 1.18
 
-require github.com/hashicorp/terraform-provider-aws v1.60.1-0.20220311191613-392d1c1db00c // v4.5.0
+require github.com/hashicorp/terraform-provider-aws v1.60.1-0.20221006134204-b94b8d2ee1ce // v4.34.0
```

Comment:
OMG I just spent several minutes trying to figure out why we were downgrading this because in my mind 4.34 < 4.5 :facepalm: